xknx/remote_value/remote_value_control.py

```diff
@@ -0,0 +1,54 @@
+"""
+Module for managing a control remote value.
+
+Examples are switching commands with priority control, relative dimming or blinds control commands.
+DPT 2.yyy and DPT 3.yyy
+"""
+from xknx.dpt import DPTBase, DPTBinary
+from xknx.exceptions import ConversionError
+
+from .remote_value import RemoteValue
+
+
+class RemoteValueControl(RemoteValue):
+    """Abstraction for remote value used for controling."""
+
+    def __init__(
+        self,
+        xknx,
+        group_address=None,
+        group_address_state=None,
+        value_type=None,
+        device_name=None,
+        after_update_cb=None,
+        invert=False,
+    ):
+        """Initialize control remote value."""
+        # pylint: disable=too-many-arguments
+        super().__init__(
+            xknx,
+            group_address,
+            group_address_state,
+            device_name=device_name,
+            after_update_cb=after_update_cb,
+        )
+        self.invert = invert
+        # pylint: disable=too-many-arguments
+        _dpt_class = DPTBase.parse_transcoder(value_type)
```

Comment:
```suggestion
        _dpt_class = DPTBase.parse_transcoder(value_type)
```
duplicate of line 27